videostore/utils.c

```diff
@@ -0,0 +1,15 @@
+#include "utils.h"
+
+int64_t get_video_duration(const char *filename) {
```

Comment:
this should be the signature. Let's be consistent about returning only errors as return values & using out params for returning data
```
int get_video_duration(
int64_t *duration, // OUT
const char *filename // IN
)
```

Reply:
when you get this working please write a c program to exercise it and run it through valgrind to confirm it doesn't leak

Reply:
Created c entrypoint cmd and added valgrind results to description.